### ISSUE TITLE
[iOS] Keystone icon is used for the imported key during import key flow

### ISSUE THREAD

**Issue by liliya-soroka** [safe-global/safe-ios#2942]
**Bug description**
Keystone icon is used for the imported key during the import key flow
**Steps To Reproduce**
1. start import key flow from Owner keys page

Current result: Keystone icon is used instead of import key 
**Expected Result**
Make sure that the correct icon is used during import key flow
**Screenshots**
<img width="369" alt="image" src="https://user-images.githubusercontent.com/338622/222142669-996de14d-3388-4caf-b5ad-914ef96b9175.png">

**Device & App version**
 - Device/OS: iPhone 11/15.0
 - App version: 3.18.2 / 3.18.0/3.19.0
 
**Environment**
prod
**Additional info**
(Optional)

**Comment by MouazAlzahabi** [comment]
@liliya-soroka I can generate this issue on main also

**Comment by liliya-soroka** [comment]
@MouazAlzahabi  Yes, it's not a 3.18.2 regression. We have had the issue for some time. That is why it's minor
